src/components/AddItem/AddItem.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { TPAComponentProps } from '../../types';
+
+import { AddItem as WSRAddItem } from 'wix-style-react';
```

Comment:
I'm not sure I understand this part.
Do all `wix-style-react` consumers need to add this build step?

Reply:
you must change the import path, otherwise the cherry-picking will not work (you will get the entire package in your bundle).

Regarding the build step - no, consumers don't need to do it in the project.
It's the library's responsibility to make sure it supports esm all the way (including in its node_modules).
Consumers will simply use webpack which knows how to handle commonjs/esm, given that the node modules are created correctly

Reply:
ok, so these are two different things, right?
1. import directly from `dist`
2. add the build step for the stylable files

Reply:
Maybe @EgoziE can give us a short explanation tomorrow on what this build step does?